scripts/supercloud/kill_all.py

```diff
@@ -0,0 +1,28 @@
+"""Script to kill all experiments running on supercloud for a given user.
+
+Runs scancel -u <user> on supercloud.
```

Comment:
lol, i really don't think it's that hard to just log into supercloud and run scancel... do we really need a local script for it?

do you not already have aliases + ssh keys set up for logging into supercloud? i just run `supercloud`, `supercloudtom`, `supercloudnjk`, etc. takes 1 second to log in

Reply:
yeah i have those aliases, i'm doing this so that
1. the scripts mirror the openstack ones
2. in a perfect world, users who are not me could use these scripts without ever having to directly ssh into the machines